drink/src/chain_api.rs

```diff
@@ -74,27 +78,28 @@ pub trait ChainApi<R: Runtime> {
 }
 
 impl<R: Runtime> ChainApi<R> for Sandbox<R> {
-    fn current_height(&mut self) -> u64 {
+    fn current_height(&mut self) -> BlockNumberFor<R> {
         self.externalities
             .execute_with(|| frame_system::Pallet::<R>::block_number())
     }
 
-    fn build_block(&mut self) -> DrinkResult<u64> {
+    fn build_block(&mut self) -> DrinkResult<BlockNumberFor<R>> {
         let current_block = self.current_height();
         self.externalities.execute_with(|| {
             let block_hash = R::finalize_block(current_block).map_err(Error::BlockFinalize)?;
-            R::initialize_block(current_block + 1, block_hash).map_err(Error::BlockInitialize)?;
-            Ok(current_block + 1)
+            let new_number = current_block + BlockNumberFor::<R>::from(1u32);
```

Comment:
consider change from `BlockNumberFor::<R>::from(1u32)`  to `BlockNumberFor<R>::one()`, also consider adding method `next/inc` on `BlockNumberFor<R>` as it is quite common pattern here

Reply:
used `::one()`; can't add method since it is a type alias for a foreign type; but fortunately it has `saturating_inc` - applied